web-console/developer/pipelines/pipelines.md

```diff
@@ -1,7 +1,191 @@
 # Pipelines
-The Pipelines page is a primary nav item in the OpenShift Developer perspective.
+- The Pipelines page is a primary navigation item in the OpenShift Developer perspective.  
+- It is only available when the pipelines operator is installed.
+- This is Tech Preview in 4.2, and we need to use the Tech Preview badge on associated pages.
 
-Design documentation for the Pipelines section can be found [here](https://docs.google.com/document/d/1L5doS4doZiG5QfCsyaiH6osiJkV9KNnPkzofPwp_W6c/edit?usp=sharing)
+## No projects exist
+When no project exist, clicking Pipelines will show an empty state:
 
-## Note
-Links to google documents are temporary until the documentation is converted to markdown files and added to this design repository.
\ No newline at end of file
+## All Projects in project selector
+When a project is not in context, clicking Pipelines will prompt the user to select from a list of projects:
```

Comment:
This is more broad than just this PR but wanted to ask: when we have lists of projects for a resource like pipelines, builds, etc, should there be an easy way to see only projects that have the resource rather than all projects (or as the default view)?